Updated to latest version of requirejs in webjars

isuftin opened the pull request:
Also updated plugin versions

isuftin committed:
Updated to latest version of requirejs in webjars

Also updated plugin versions

jamesward commented:
Is there a new version of this artifact?  If so you'll need to also update:
https://github.com/isuftin/requirejs-text/blob/master/pom.xml#L57
And maybe: https://github.com/isuftin/requirejs-text/blob/master/pom.xml#L14

isuftin committed:
Updated to include latest requirejs and change 2.0.14 snapshot version to 2.0.14-1

isuftin commented:
@jamesward There's no new version of the base artifact (requirejs-text) but I've updated the pom to dependon the latest version of requirejs.  I've updated the snapshot version to a include a '-1'
